WordPress/src/main/res/layout/media_picker_fragment.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout
```

Comment:
I think the font size of the empty TextView is a little too large and that wording is kind of odd. I'd remove "Why not capture some?" and just show "No videos" or "No images."

![device-2015-04-07-050030](https://cloud.githubusercontent.com/assets/3903757/7020000/7e59a4b4-dce3-11e4-8a0b-f002309427d4.png)